react/features/toolbox/components/web/ReactionEmoji.js

```diff
@@ -0,0 +1,96 @@
+// @flow
+
+import React, { Component } from 'react';
+
+import { connect } from '../../../base/redux';
+import { removeReaction } from '../../actions.any';
+import { REACTIONS } from '../../constants';
+
+type Props = {
+
+    /**
+     * Reaction to be displayed.
+     */
+    reaction: string,
+
+    /**
+     * Id of the reaction.
+     */
+    uid: Number,
+
+    /**
+     * Removes reaction from redux state.
+     */
+    removeReaction: Function,
+
+    /**
+     * Index of the reaction in the queue.
+     */
+    index: number
+};
+
+type State = {
+
+    /**
+     * Index of CSS animation. Number between 0-20.
+     */
+    index: number
+}
+
+
+/**
+ * Used to display animated reactions.
+ *
+ * @returns {ReactElement}
+ */
+class ReactionEmoji extends Component<Props, State> {
+    /**
+     * Initializes a new {@code ReactionEmoji} instance.
+     *
+     * @param {Props} props - The read-only React {@code Component} props with
+     * which the new instance is to be initialized.
+     */
+    constructor(props: Props) {
+        super(props);
+
+        this.state = {
+            index: props.index % 21
```

Comment:
Because the index from the props can change during the lifetime of the component and we need an animationIndex that is constant for the entire lifetime